packages/styles/components/Tooltip.module.scss

```diff
@@ -76,23 +78,24 @@ $arrow-size: 10px;
           left: 42%;
           position: absolute;
           top: 127%;
-          transform: translateX(-50%);
-          transform: translateY(-50%);
+          transform: translateX(-$coordinate-space);
+          transform: translateY(-$coordinate-space);
```

Comment:
We should to use `transform(-$coordinate-space, -$coordinate-space)` or `transform: translateY(-$coordinate-space);` will be overwrite `translateX(-$coordinate-space);`

Reply:
Done, removed code:  transform: translateX(-$coordinate-space);